formats/finder_email_signup.jsonnet

```diff
@@ -178,6 +178,14 @@
             },
           },
         },
+        join_facets_with: {
```

Comment:
I'm wondering if `join_facets_with` could be named `combine_mode` to make it consistent with the finder content item (and also optimise for deletion later on)?